server/mdm/assets/assets.go

```diff
@@ -101,25 +101,30 @@ func ABMToken(ctx context.Context, ds fleet.MDMAssetRetriever, abmOrgName string
 		return nil, fmt.Errorf("parsing ABM certificate: %w", err)
 	}
 
-	return DecryptRawABMToken(
-		assets[fleet.MDMAssetABMTokenDeprecated].Value, //abmTok.EncryptedToken,
+	oAuthTok, _, err := DecryptRawABMToken(
+		assets[fleet.MDMAssetABMTokenDeprecated].Value, // abmTok.EncryptedToken,
```

Comment:
This shouldn't be used anymore (after DB migration, the ABM token will not be in the MDM config assets). What may be missing to get rid of it is the TODO I noted above? https://github.com/fleetdm/fleet/pull/21419/files#diff-34a4a4ab071895cc6bcc79a0bd67461497014138edcd6271676c6d604cc615c6R85